vitrina/templates/component/hero.html

```diff
@@ -20,7 +20,7 @@ <h2 class="subtitle site-subtitle">
                     <div class="has-dark-blue-background hero-column">
                         <p class="is-size-5">{% translate "Nerandate duomenų rinkinio?" %}</p>
                         <p class="is-size-5">{% translate "O gal esamas duomenų rinkinys neatitinka lūkesčių?" %}</p>
-                        <a href="/requests/request" class="button is-primary is-normal m-t-md is-size-6-mobile">
+                        <a href="{% url 'request-create' %}" class="button is-primary is-normal m-t-md is-size-6-mobile">
```

Comment:
We need a test, that clicks on this button, to make sure, that this feature works.